include/aws_jobs.h

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/**@file aws_jobs.h
```

Comment:
Please ensure that the syntax used here generates correct doc output.

Look at `include/spm.h` on how to define group which is used when linking in API docs to the rst.

Reply:
Added